Assets/00_Content/Prefabs/Interactables/BeerBarrel.prefab

```diff
@@ -63,7 +63,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: ab5ac90e8d6ee7449933c889df334329, type: 3}
```

Comment:
I find it a bit odd that the solo carry idle only uses one hand.

Reply:
Fixed in master.